mitmproxy/tools/web/app.py

```diff
@@ -470,6 +479,16 @@ def put(self):
             raise APIError(400, f"{err}")
 
 
+class CommandArguments(RequestHandler):
+    def get(self):
+        arguments = {}
+        for (name, command) in self.master.commands.commands.items():
+            arguments[name] = []
+            for parameter in command.parameters:
+                arguments[name].append(parameter.name)
+        self.write(arguments)
```

Comment:
This looks good overall. Some ideas:

1. We can move this method in the `Commands` class so that we only need one route. (`GET /commands` to access the available options, `POST /commands` to issue one)
2. Let's append `{"name": parameter.name}` instead of `parameter.name`. We may want to add the parameter type here at some point.
3. `arguments` should probably be named `commands`?

Reply:
Done!